connector-runtime/spring-boot-starter-camunda-connectors/src/test/java/io/camunda/connector/runtime/inbound/WebhookControllerPlainJavaTests.java

```diff
@@ -163,7 +166,7 @@ public static InboundConnectorContextImpl buildContext(InboundConnectorDetails d
   public static InboundConnectorDetails webhookDefinition(
       String bpmnProcessId, int version, String path) {
     return new InboundConnectorDetails(
-        UUID.randomUUID().toString(),
+        bpmnProcessId + version + path,
```

Comment:
Where will it be used? If it's user exposed, maybe consider concatenating with dashes or something 🤔 . Just a suggestion.

```suggestion
        bpmnProcessId + "-" + version + "-" + path,
```

Reply:
It's a mock value for tests only 🙂